packages/suite/src/components/suite/skeletons.ts

```diff
@@ -48,14 +50,15 @@ const shimmerEffect = css`
     background: linear-gradient(
         90deg,
         ${({ theme }) =>
-            `${theme.GRADIENT_SKELETON_START}, ${theme.BG_GREY_ALT}, ${theme.GRADIENT_SKELETON_START}`}
+            `${theme.gradientNeutralBottomFadeSurfaceElevation1Start}, ${theme.gradientNeutralBottomFadeSurfaceElevation1End}, ${theme.gradientNeutralBottomFadeSurfaceElevation1Start}`}
     );
     background-size: 200%;
 `;
 
 type SkeletonBaseProps = {
     background?: string;
     animate?: boolean;
+    elevation?: 0 | 1 | 2 | 3;
```

Comment:
how about making it a type somewhere in the components package? And I feel like the skeletons can even be moved there as well!

Reply:
Good idea! I will do it

Reply:
Just realized it's already [used](https://github.com/trezor/trezor-suite/blob/414cf533c4bc6199e1b6193f90b412562f54a4bf/packages/components/src/components/Card/Card.tsx#L29) in the `Card`!

Reply:
So basically we need multiple elevations for borders, backgrounds, box shadows etc. Right?

Reply:
Yes, this all is going to be handled by the [`ElevationContext`](https://github.com/trezor/trezor-suite/issues/10140), once we are done with the app. The `Card` kinda shows how it will be done but it all will be automated with `ElevationContext` going forward.

Reply:
@dahaca Done (btw sorry I amended commit 🤦‍♂️)

Reply:
No problemo! All is sound now so it's gut ☮️